_posts/2025-01-30-one-click-deepseek-integration.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title:  "Instant DeepSeek: One-Click Activation with OpenSearch"
+authors:
+   - kazabdu
+   - minalsha
+   - seanzheng
+   - amitgalitz
+date: 2025-01-30
+categories:
+  - technical-posts
+meta_keywords: OpenSearch DeepSeek integration, LLM integration, RAG, AI search, Flow Framework, machine learning, natural language processing, open-source LLM
+meta_description: Learn how to enable DeepSeek LLM integration in OpenSearch with just one API call using the Flow Framework plugin, simplifying the setup process for conversational search and RAG.
+---
+
+
+In the ([previous blog](https://opensearch.org/blog/OpenSearch-Now-Supports-DeepSeek-Chat-Models/)) post we shared that OpenSearch supports DeepSeek LLM. This blogpost talks about bringing simplicity in provisioning DeepSeek LLM integration with just one API call using OpenSearch Flow Framework plugin.  We’ve provided automated templates, enabling you to create connectors, register models, deploy them, and register agents and tools through a single API call. This eliminates the complexity of calling multiple APIs and orchestrating setups based on the responses.
+
+
+### Without Flow Framework plugin
+
+In the ([previous blog](https://opensearch.org/blog/OpenSearch-Now-Supports-DeepSeek-Chat-Models/)) blog post, setting up DeepSeek model or any LLM model involves four separate API calls:
+
+1. Create a connector for DeepSeek model
+2. Create a model group
+3. Register the model using the connector ID
+4. Create a search pipeline for RAG
+
+
+Via OpenSearch Flow Framework plugin, we are simplifying this experience by enabling you to make a single API call. Let’s demonstrate how the plugin simplifies this process using the preceding conversational search example.
+
+
+### With the Flow Framework plugin
+
+In the following example, you will configure the  conversational_search_with_llm_deploy  workflow template to implement RAG with DeepSeek in OpenSearch. The workflow created using this template performs following configuration steps:
```

Comment:
remove 1 spacing both before and after "conversational_search_with_llm_deploy". Also, bold the template name

Reply:
should be as:  In the following example, you will configure the **conversational_search_with_llm_deploy** workflow template to implement RAG with DeepSeek in OpenSearch. The workflow created using this template performs following configuration steps: